tools/devtool

```diff
@@ -802,6 +806,14 @@ cmd_test() {
     # do not leave behind env.list file
     rm env.list
 
+    # archive everything in the `test_result` to speed up upload/download
+    # to s3 if we are in CI
+    if [ $do_archive != 0 ] && [ -n "$BUILDKITE" ] && [ "$BUILDKITE" = "true" ]; then
+      tar -czf data.tar -C test_results .
```

Comment:
I thought compressed tars were usually called ".tar.gz". Could less confusing for manual processing.